main.py

```
@@ -0,0 +1,63 @@
+
+    try:
+        # 이름이 없는 경우
+        if not tmp_db:
+            return {"message": "등록된 이름이 없습니다", "names": name_list}
+        # 성공!
+        return {"message": "이름 목록을 가져왔습니다", "names": name_list}
+    except Exception as e:
+        # 예상치 못한 오류가 발생한 경우
+        raise HTTPException(
+            status_code=500, detail="서버 오류가 발생했습니다 : " + str(e)
+        )
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)